mne/stats/regression.py

```diff
@@ -246,13 +246,12 @@ def linear_regression_raw(raw, events, event_id=None, tmin=-.1, tmax=1,
     """
 
     if isinstance(solver, string_types):
-        if solver == 'pinv':
-            fast_dot = _get_fast_dot()
+        if solver == 'cholesky':
+            def solver(X, y):
+                a = (X.T * X).toarray()
```

Comment:
This is fine then, but please add a comment saying X is a matrix object
therefore * means dot